openstef/metrics/metrics.py

```diff
@@ -362,7 +362,7 @@ def xgb_quantile_obj(preds, dmatrix, quantile=0.2):
     left_mask = errors < 0
     right_mask = errors > 0
 
-    grad = -quantile * left_mask + (1 - quantile) * right_mask
+    grad = (quantile * left_mask + (1 - quantile) * right_mask) * errors
```

Comment:
Great!
Maybe add a little docstring explaing the resizing by errors (since this is different from the original, referenced loss function

Reply:
Good idea, I will add it.